perun-base/src/main/java/cz/metacentrum/perun/audit/events/MailManagerEvents/InvitationSentEvent.java

```diff
@@ -0,0 +1,45 @@
+package cz.metacentrum.perun.audit.events.MailManagerEvents;
+
+import cz.metacentrum.perun.audit.events.AuditEvent;
+import cz.metacentrum.perun.core.api.User;
+
+public class InvitationSentEvent extends AuditEvent {
+
+	private User sender;
+	private String email;
+	private String language;
+	private String message;
+
+	public InvitationSentEvent() {
+	}
+
+	public InvitationSentEvent(User user, String mail, String language) {
```

Comment:
Can you please add also VO and Group object to the event ? We need to know, into which entity was somebody invited.